types/rlp_unmarshal_storage.go

```diff
@@ -58,7 +80,22 @@ func (b *Body) UnmarshalRLPFrom(p *fastrlp.Parser, v *fastrlp.Value) error {
 }
 
 func (t *Transaction) UnmarshalStoreRLP(input []byte) error {
-	return UnmarshalRlp(t.UnmarshalStoreRLPFrom, input)
+	txType := TxTypeLegacy
+
+	if len(input) > 0 && input[0] <= 0x7F {
```

Comment:
What is this magic number for input? Please extract it out and make it clear

Reply:
Added at b4008cc9699c1c0f45193b28975cd5fba4483a95